deploy/runtime/ui/Fuel Cell Dispatch.txt

```diff
@@ -426,20 +426,55 @@ equations{ 'fuelcell_dispatch'} = define(){
 	}
 	return dispatch;
 };
-418
-on_load{'Fuel Cell Dispatch'} = define() {
-	on_change{'fuelcell_dispatch_choice'}(); 
-	toggle_dispatch();
+2116
+on_load{'Fuel Cell Dispatch'} = define() 
+{
+	enable_fc_dispatch_options();
 };
-on_change{'fuelcell_dispatch_choice'} = define(){
+
+on_change{'fuelcell_dispatch_choice'} = define()
+{
+	enable_fc_dispatch_options();
+	toggle_dispatch_fc(); // from Fuel Cell Dispatch Manual so manual dispatch options update
+	if ( value('fuelcell_dispatch_choice') == 2 && value('batt_dispatch_choice_ui') != 3 )
+	{
+		fuel_cell_manual_dispatch();
+	}
+};
+
+function enable_fc_dispatch_options()
+{
 	enable('fuelcell_fixed_pct', value('fuelcell_dispatch_choice') == 0);
 	enable('fuelcell_dispatch_input', value('fuelcell_dispatch_choice') == 3);
 	enable('fuelcell_dispatch_input_units', value('fuelcell_dispatch_choice') == 3);
+}
 
-	toggle_dispatch();
-}; 
-
-
+function fuel_cell_manual_dispatch()
```

Comment:
If you go from "Enable Battery" on the Battery Storage page to "No Battery" with "Manual Dispatch" selected for the battery dispatch option, "Manual Dispatch" can be selected for the Fuel Cell without any prompts or battery enabling. Is this intended behavior?

Reply:
Good catch. I fixed that in the latest commit -- the UI callback checks for manual dispatch should work for BTM and FOM.